find_attitude/make_distances.py

```diff
@@ -0,0 +1,140 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import argparse
+import os
+
+import agasc
+import astropy.units as u
+import astropy_healpix as hp
+import numpy as np
+import tables
+from astropy.coordinates import SkyCoord
+from astropy.table import Table
+
+MIN_ACA_DIST = 25.0 * u.arcsec  # 25 arcsec min separation (FFS won't find closer)
+MAX_ACA_DIST = 2.0 * u.deg  # degrees corner to corner (actual max is 7148 arcsec)
+MAX_MAG = 10.0  # Max mag to include in the distances file
+DATE_DISTANCES = "2025:001"  # Date for proper motion correction in distances
+HEALPIX_NSIDE = 16
+HEALPIX_ORDER = "nested"
+SKA = os.environ["SKA"]
```

Comment:
What is this here for?

Reply:
Looks like a leftover, but fixing it is likely not worth the trouble of re-testing.